content/en/docs/components/pipelines/concepts/metadata.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "ML Metadata"
+description = "Conceptual overview about Metadata in Kubeflow Pipelines"
+weight = 90
+                    
++++
+
+**Note:** Kubeflow Pipelines has been migrated from [kubeflow/metadata](https://github.com/kubeflow/metadata)
```

Comment:
This reads as though Kubeflow pipelines was migrated from the metadata repo, when I believe you mean that Kubeflow Pipelines no longer uses kubeflow/metadata and now uses google/ml-metadata instead?

Also can we make this active voice?

Reply:
Good catch, I have updated the working to use active voice.